tests/test_logger.py

```
@@ -35,3 +35,25 @@ def test_log_formats_message(
+    logger = FemtoLogger("core")
+    with pytest.raises(ValueError):
+        logger.set_level("BAD")
+
+
+def test_is_enabled_for_invalid_raises() -> None:
+    logger = FemtoLogger("core")
+    with pytest.raises(ValueError):
+        logger.is_enabled_for("BAD")
+
+
+def test_invalid_level_defaults_to_info() -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)